### ISSUE TITLE
Display warning in the react UI

### ISSUE THREAD

**Issue by roidelapluie** [prometheus/prometheus#8030]
## Proposal
**Use case. Why is this important?**

The Classic UI displays PromQL warning when running a query:

![class](https://user-images.githubusercontent.com/291750/95444710-66ba6700-095e-11eb-8ace-e922e335f84a.png)

We could have the same in the new UI:

![new](https://user-images.githubusercontent.com/291750/95444733-6ae68480-095e-11eb-938f-4e3d0cde88e1.png)

Prometheus.yml to reproduce:

```
remote_read:
  - url: http://127.0.0.1:9090/test
    read_recent: true
```

**Comment by zerefwayne** [comment]
Hi @roidelapluie, I would like to solve this.

**Comment by roidelapluie** [comment]
> Hi @roidelapluie, I would like to solve this.

That would be awesome :) 👍

**Comment by aSquare14** [comment]
Can I work on this @roidelapluie ?

**Comment by roidelapluie** [comment]
> Can I work on this @roidelapluie ?

I missed your message 😯 sorry!

Fixed in #8427
